src/ert/_c_wrappers/job_queue/job_queue_node.py

```diff
@@ -170,13 +171,26 @@ def _job_monitor(self, driver, pool_sema, max_submit):
 
         current_status = self.refresh_status(driver)
 
+        initial_sleep_seconds = 1
+        time_sleep_seconds = initial_sleep_seconds
+        max_sleep_seconds = 30
+        use_random_sleep_offset = False
         while self.is_running(current_status):
             if (
                 self._start_time is None
                 and current_status == JobStatusType.JOB_QUEUE_RUNNING
             ):
                 self._start_time = time.time()
-            time.sleep(1)
+            if (
+                self._start_time is not None
+                and time_sleep_seconds != max_sleep_seconds
+                and time.time() - self._start_time > 30
+            ):
+                time_sleep_seconds = max_sleep_seconds
+                use_random_sleep_offset = True
+            time.sleep(
+                time_sleep_seconds + use_random_sleep_offset * random.randint(-5, 5)
```

Comment:
I like this step, which desyncronize threads!

Reply:
thank you : )